crates/cli/src/app.rs

```diff
@@ -289,10 +289,15 @@ pub enum Commands {
     )]
     Check {
         #[arg(help = "List of project IDs to explicitly check")]
+        #[clap(group = "projects")]
```

Comment:
what does this do exactly?

Reply:
This makes `[ID]s` and `--all` mutually exclusive so that they can not be used together.

Reply:
<img width="673" alt="image" src="https://user-images.githubusercontent.com/60938164/198932306-d5971c98-6fb8-419b-858f-545092a584ff.png">

Reply:
Awesome